src/plugins/saved_objects_management/public/management_section/mount_section.tsx

```diff
@@ -32,35 +32,38 @@ import React, { lazy, Suspense } from 'react';
 import ReactDOM from 'react-dom';
 import { Router, Switch, Route } from 'react-router-dom';
 import { I18nProvider } from '@osd/i18n/react';
-import { i18n } from '@osd/i18n';
 import { EuiLoadingSpinner } from '@elastic/eui';
-import { CoreSetup } from 'src/core/public';
-import { ManagementAppMountParams } from '../../../management/public';
+import { AppMountParameters, CoreSetup } from 'src/core/public';
+import { ManagementAppMountParams } from 'src/plugins/management/public';
+import { PageWrapper } from './page_wrapper';
 import { StartDependencies, SavedObjectsManagementPluginStart } from '../plugin';
 import { ISavedObjectsManagementServiceRegistry } from '../services';
 import { getAllowedTypes } from './../lib';
 
 interface MountParams {
   core: CoreSetup<StartDependencies, SavedObjectsManagementPluginStart>;
   serviceRegistry: ISavedObjectsManagementServiceRegistry;
-  mountParams: ManagementAppMountParams;
+  mountParams?: ManagementAppMountParams;
+  appMountParams?: AppMountParameters;
```

Comment:
Why do we need to add `appMountParams`? Seems the new added `appMountParams` was used in `const usedMountParams = mountParams || appMountParams || ({} as ManagementAppMountParams);` only. Could we use `mountParams` instead?

Reply:
Seems that `mountManagementSection` function is only called in `src/plugins/saved_objects_management/public/plugin.ts` with passed by `appMountParams` instead of `mountParams`,  right? @SuZhou-Joe

Reply:
Yes, `mountParams` is deprecated and we should only use `appMountParams`.